ios/ReferencedAssetLoader.swift

```diff
@@ -249,4 +471,8 @@ final class ReferencedAssetLoader {
       return true
     }
   }
+
+  func dispose() {
```

Comment:
Where is this dispose is called from?!

Reply:
hmm actually looks like it never is. Guess its a bug

Reply:
@mfazekas I was also seeing a similar issue on Android

Reply:
I removed the disposal on android and ios, and rebased on main